homeassistant/components/imap/config_flow.py

```diff
@@ -87,6 +88,7 @@
         cv.positive_int,
         vol.Range(min=DEFAULT_MAX_MESSAGE_SIZE, max=MAX_MESSAGE_SIZE_LIMIT),
     ),
+    vol.Optional(CONF_ENABLE_PUSH, default=False): BOOLEAN_SELECTOR,
```

Comment:
Should this default to True like it does in the main path?

Reply:
Thnx, yes, of course